ui/src/config/urls.ts

```diff
@@ -106,6 +106,13 @@ const urls = {
         keywords: [],
         canonical: `${base.host}/account`
     },
+    myBookmarks: {
+        path: '/my-bookmarks',
+        title: `${base.title}`,
+        description: '',
+        keywords: [],
+        canonical: `${base.host}/authors`
```

Comment:
Think this needs updating to `${base.host}/my-bookmarks`

Reply:
good spot - I'll amend